liquidctl/driver/kraken3.py

```diff
@@ -695,6 +688,32 @@ def get_status(self, **kwargs):
             (_STATUS_FAN_DUTY, msg[25], "%"),
         ]
 
+    def _get_status_from_hwmon(self):
+        return [
+            (_STATUS_TEMPERATURE, self._hwmon.read_int("temp1_input") * 1e-3, "°C"),
+            (_STATUS_PUMP_SPEED, self._hwmon.read_int("fan1_input"), "rpm"),
+            (_STATUS_PUMP_DUTY, self._hwmon.read_int("pwm1") * 100.0 / 255, "%"),
+            (_STATUS_FAN_SPEED, self._hwmon.read_int("fan2_input"), "rpm"),
+            (_STATUS_FAN_DUTY, self._hwmon.read_int("pwm2") * 100.0 / 255, "%"),
+        ]
+
+    def get_status(self, direct_access=False, **kwargs):
```

Comment:
If I'm not mistaken, this is an unnecessary override.  Can't the Z class just use the base X class method?

Reply:
Of course, just pushed that commit.